### ISSUE TITLE
Aux loss decreases and then increases

### ISSUE THREAD

**Issue by eadstry** [InterDigitalInc/CompressAI#322]
First of all, I would like to thank you for your valuable work!
When I train the model, I find that all other losses are normal, but the aux loss goes down and then goes up:
![image](https://github.com/user-attachments/assets/970db6cd-543c-436f-88da-6ae7a285e636)
I've read #https://github.com/InterDigitalInc/CompressAI/issues/148, but gradient clipping doesn't work.
After a few hundred epochs of training, I looked at the quantiles:
![image](https://github.com/user-attachments/assets/67726827-2f37-48f5-8ec1-be0c15d5840a)

and aux loss up to 37869.1953.
But when I look at the gradient, I see that the gradient at the quantiles[1] is higher than the quantiles[2] and quantiles[0]:
![image](https://github.com/user-attachments/assets/8441833a-3efe-482f-9893-6194c2eabc58)
Based on the above, I have three questions:
1. quantiles seem to converge despite the large aux loss. Does this affect the performance when actually compressing?
2. I observe that the difference between the upper and lower quantiles and self.target is larger, but the gradient at the median is larger. Is this the reason why the aux is increasing in my training?
3. What measures should I take to ensure that the aux keeps decreasing?

Looking forward to your reply!
